DTS-STN/Scrum-Poker
# Dums 96 custom cards

## [DUMS-96 - Render cards on room page based on a query from the server](https://jira-dev.bdm-dev.dts-stn.com/browse/DUMS-96) (Jira Issue)

### Description

This PR builds off of #101. The card list is now filtered through based on the values coming from the server (which were chosen on the home page).

### What to test for/How to test

1. Pull in branch
2. Type `npm run dev`
3. Select any combination of cards on the home page and create a room
4. Once on the room page, you'll see that the cards you selected are the ones that are showing
5. Select a card and and hit clear, the card list should stay the same.
